react-native-theoplayer.podspec

```diff
@@ -25,15 +25,18 @@ Pod::Spec.new do |s|
   s.platforms    = { :ios => "13.4", :tvos => "13.4" }
   s.source       = { :git => "https://www.theoplayer.com/.git", :tag => "#{s.version}" }
 
-  s.source_files = 'ios/*.{h,m,swift}', 'ios/ads/*.swift', 'ios/casting/*.swift', 'ios/contentprotection/*.swift', 'ios/pip/*.swift', 'ios/backgroundAudio/*.swift', 'ios/cache/*.swift', 'ios/sideloadedMetadata/*.swift', 'ios/eventBroadcasting/*.swift' , 'ios/ui/*.swift', 'ios/presentationMode/*.swift', 'ios/viewController/*.swift'
+  s.source_files = 'ios/*.{h,m,swift}', 'ios/ads/*.swift', 'ios/casting/*.swift', 'ios/contentprotection/*.swift', 'ios/pip/*.swift', 'ios/backgroundAudio/*.swift', 'ios/cache/*.swift', 'ios/sideloadedMetadata/*.swift', 'ios/eventBroadcasting/*.swift' , 'ios/ui/*.swift', 'ios/presentationMode/*.swift', 'ios/viewController/*.swift', 'ios/theolive/*.swift'
   s.resources = ['ios/*.css']
 
   # ReactNative Dependency
   s.dependency "React-Core"
 
   # THEOplayer core Dependency
   puts "Adding THEOplayerSDK-core"
-  s.dependency "THEOplayerSDK-core", "~> 8.3"
+  s.dependency "THEOplayerSDK-core", "~> 8.5"
+  puts "Adding THEOplayer-Integration-TheoLive"
+  s.dependency "THEOliveSDK", "~> 3.18.0"
+  s.dependency "THEOplayer-Integration-TheoLive", "~> 8.5"
```

Comment:
you can use 8.6.0 and above, and I think you can omit THEOliveSDK, because the [THEOplayer-Integration-TheoLive](
https://github.com/CocoaPods/Specs/commit/a2cc45da84cc7e4dfec9e4d2c502a43872156dfe#diff-c3b46785d74f03af227a4237a9ec0194a629a18378030539427e60427f3d4db2R2) will pull it in.